src/app/modules/main/wallet_section/send/network_model.nim

```diff
@@ -275,6 +275,14 @@ QtObject:
         self.items[i].isEnabled = not disabled
         self.dataChanged(index, index, @[ModelRole.IsEnabled.int])
 
+  proc setEnabledFromChains*(self: NetworkModel, chainId: int) {.slot.} =
```

Comment:
if the chains we can use are restricted, we should only display collectibles for those chains in the list. Could you update https://github.com/status-im/status-desktop/pull/12051/files#diff-1702286f2fd9311b35a8dcbc966e5d243ccd2d1dd351959048736a63c6710312R317 ?

Reply:
Hey I am only trying to set the chain the collectible is on to enabled=true
So this selection is based on what the collectible properties are and not vice versa, makes sense?

Reply:
ohhh yeah that makes sense. Perhaps the method name is not specific enough in that case, maybe something like `setRouteEnabledFromChains`?